File: C#/백준/Bronze/10801. 카드게임/카드게임.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int cnt = 0;
            for (int i=0;i<10;i++){
                if (aa[i]==bb[i]){
                    continue;
                }
                else if(aa[i]>bb[i]){
                    cnt++;
                }
                else{
                    cnt--;
                }
            }
            if (cnt==0){
                Console.WriteLine("D");
            }
            else{
                Console.WriteLine((cnt>0)?"A":"B");
            }
        }
    }
}

File: Python/백준/Bronze/10801. 카드게임/카드게임.py
aa = list(map(int,input().split()))
bb = list(map(int,input().split()))
s1 = 0
s2 = 0
for i in range(10):
    if aa[i]>bb[i]:
        s1+=1
    elif aa[i]==bb[i]:
        continue
    else:
        s2+=1
if s1==s2:
    print('D')
elif s1>s2:
    print('A')
else:
    print('B')